### PR TITLE
test

### PR DESCRIPTION
**Pull request by josovva**

### CODE REVIEW

**Review by josovva — COMMENTED** [review]
XD

File: README.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ Web application with backend written in C# with .NET 6 and Angular used on front
 ## Technologies
 
 ## Authors
+Asia Dejwid i Kubuś
```
Comment (kubaurban) [inline README.md:12]: WTF

File: README.md [hunk 2]
```diff
@@ -9,3 +9,4 @@ Web application with backend written in C# with .NET 6 and Angular used on front
 ## Technologies
```
Comment (kubaurban) [inline README.md:9]: Pretty good one

**Review by kubaurban — CHANGES_REQUESTED** [review]
Lets view it again... @josovva

**Review by kubaurban — APPROVED** [review]
Muah

**Comment by kubaurban** [comment]
just a comment

**Pull request closed without merging**